txwrapplog.py

```diff
@@ -22,8 +23,10 @@
 }
 
 
-def wrapp_observer(output):
+def wrapp_observer(output, service=None, host=None):
     json = jsonFileLogObserver(output, recordSeparator='')
+    service = service or os.environ.get('SERVICE_NAME')
```

Comment:
Why not run this assignment outside and the make the function definition always accept service and host ?